### ISSUE TITLE
Make AeroTab it's own repository

### ISSUE THREAD

**Issue by gold2718** [NorESMhub/CAM#113]
### Discussed in https://github.com/NorESMhub/CAM/discussions/104

<div type='discussions-op-text'>

<sup>Originally posted by **Ovewh** October 20, 2023</sup>
AeroTab, which is the code that generates the optical tables used in Oslo_Aero,  currently lives in the cam/tools directory (not in the noresm_develop branch). 
Because lacking of documentation and being hard to use, the tables we are using in Oslo_Aero have not really been look at for the past six years.

The code needs to be refactored to become more configurable and better documented, a end goal could be to make a python wrapper and create something like a pyAeroTab. In my opinion this could make AeroTab into a great tool exploring impact of e.g. aerosol processing (aging and hygroscopic growth) on size distribution and a useful tool for e.g. teaching purposes. 

TL;DR Initiate a process of refactoring AeroTab and make into its own repository. </div>

**Comment by MichaelSchulzMETNO** [comment]
Maybe not needed in 2.1?

**Comment by gold2718** [comment]
> Maybe not needed in 2.1?

Good point, thanks!. The repo [exists](https://github.com/NorESMhub/AeroTab) but so far, 2.1 does not have any CAM externals so  this is not needed.
Moving this to NorESM2.3.

**Comment by gold2718** [comment]
Implemented in #246
